src/components/guide/index.js

```diff
@@ -0,0 +1,221 @@
+import PropTypes from "prop-types";
+import TraitifyPropTypes from "lib/helpers/prop-types";
+import {Component} from "react";
+import withTraitify from "lib/with-traitify";
+import guideQuery from "lib/graphql/queries/guide";
+import {dangerousProps, listId} from "lib/helpers";
+import smallScreen from "./helpers/helpers";
+import style from "./style";
+
+class Guide extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      badges: [],
+      competencies: [],
+      displayedCompetency: {},
+      errors: [],
+      expandedIntro: false
+    };
+  }
+  static defaultProps = {assessment: null};
+  static propTypes = {
+    airbrake: PropTypes.shape({notify: PropTypes.func}).isRequired,
+    assessment: PropTypes.shape({personality_types: PropTypes.array}),
+    assessmentID: PropTypes.string.isRequired,
+    locale: PropTypes.string.isRequired,
+    translate: PropTypes.func.isRequired,
+    traitify: TraitifyPropTypes.traitify.isRequired
+  }
+  componentDidMount() {
+    this.setGuide();
+  }
+  componentDidUpdate(prevProps) {
+    if(this.props.locale !== prevProps.locale) {
+      this.setGuide();
+    }
+  }
+  adaptability(adaptability) {
+    if(!adaptability) { return; }
+    const {translate} = this.props;
+
+    return (
+      <div>
+        <h4>{translate("question_adaptability")}</h4>
+        <div>{this.stringToListItems(adaptability)}</div>
+      </div>
+    );
+  }
+  displayCompetency(competency) {
+    const competencyName = typeof (competency) === "string" ? competency : competency.target.value;
+    this.state.competencies.forEach((comp) => {
+      if(competencyName === comp.name) {
+        this.setState({displayedCompetency: comp});
+      }
+    });
+  }
+  expandedIntro(text) {
+    if(this.state.expandedIntro) {
+      return (
+        <p>{text}</p>
+      );
+    }
+  }
+  processError(errors) {
```

Comment:
Mind removing this? Since we decided to drop Airbrake from here. My bad for having you add it